NeoMathEngine/src/CPU/x86/CpuX86MathEngineVectorMathMkl.cpp

```diff
@@ -250,6 +250,39 @@ void CCpuMathEngine::vectorEltwiseLogSumExp(const CConstFloatHandle& firstHandle
 	VectorAdd(resultHandle, tempBuffer.GetHandle(), resultHandle, vectorSize);
 }
 
+void CCpuMathEngine::VectorEltwiseDivide(const CConstIntHandle& firstHandle,
+	const CConstIntHandle& secondHandle, const CIntHandle& resultHandle, int vectorSize)
+{
+	ASSERT_EXPR( firstHandle.GetMathEngine() == this );
+	ASSERT_EXPR( secondHandle.GetMathEngine() == this );
+	ASSERT_EXPR( resultHandle.GetMathEngine() == this );
+	CCpuExecutionScope scope;
+
+	const int* first = GetRaw(firstHandle);
+	const int* second = GetRaw(secondHandle);
+	int* result = GetRaw(resultHandle);
+
+#ifdef NEOML_USE_MKL // _mm_div_epi32 is a part of MKL
```

Comment:
`_mm_div_epi32` it's a part of SSE not MKL.

Reply:
Yep. It's a part of SVML (not SSE/AVX), which is an Intel's extension and it's supported by ICC and MSVC.